earthly/rust/Earthfile

```diff
@@ -69,14 +69,15 @@ rust-base:
     # Install tools we use commonly with `cargo`.
     # Note, we disable static compiles for tools, specifically, as its not required.
     # These tools are not artifacts and we do not use them in production.
-    RUN cargo install cargo-nextest --version=0.9.59 && \
+    RUN cargo install cargo-nextest --version=0.9.67 && \
         cargo install cargo-machete --version=0.6.0  && \
-        cargo install refinery_cli  --version=0.8.11 && \
-        cargo install cargo-deny    --version=0.14.3 && \
-        cargo install cargo-modules --version=0.10.2 && \
-        cargo install cargo-depgraph --version=1.5.0 && \
-        cargo install cargo-llvm-cov --version=0.5.39 && \
-        cargo install --git https://github.com/bytecodealliance/wasmtime --tag v16.0.0 verify-component-adapter
+        cargo install refinery_cli  --version=0.8.12 && \
+        cargo install cargo-deny    --version=0.14.10 && \
+        cargo install cargo-modules --version=0.13.5 && \
+        cargo install cargo-depgraph --version=1.6.0 && \
+        cargo install cargo-llvm-cov --version=0.6.4 && \
+        cargo install --git https://github.com/bytecodealliance/wasmtime --tag v17.0.0 verify-component-adapter && \
+        cargo install --git https://github.com/bytecodealliance/wit-bindgen --rev 442f005 wit-bindgen-cli
```

Comment:
don't we want to use `wasm-tools` instead of it ?

Reply:
that tool is not used yet, but probably will be in the future so I added it as well.